python/sglang/srt/mem_cache/memory_pool.py

```diff
@@ -127,27 +136,80 @@ def __init__(
         head_dim: int,
         layer_num: int,
         device: str,
+        kv_cache_dtype_str: str,
+        kvint4_groupsize: int,
+        torch_dtype: torch.dtype,
     ):
         super().__init__(size, dtype, device)
 
-        # [size, head_num, head_dim] for each layer
-        # The padded slot 0 is used for writing dummy outputs from padded tokens.
```

Comment:
keep this comment
`# The padded slot 0 is used for writing dummy outputs from padded tokens.`